### ISSUE TITLE
How can you access to resources from the current path?

### ISSUE THREAD

**Issue by dbranco** [DavideViolante/Angular-Full-Stack#99]
Hi,

I am using Visual Studio Code, but maybe this is for all editors. If I am in the client folder for example and I am editing the index.html, the editor doesn't activate the intellisense when trying:

<link rel="icon" type="image/x-icon" href="assets/favicon.ico">

For the href="..." attribute, I mean it is a little annoying.

Another example in main.ts when trying to complete:

import { AppModule } from './';

After ./ I use ctrl+space to display my local development folder and "No Suggestion" appears, it only begins to work after providing the first folder ./app

So in summary, the ctrl+space and ctrl+click has a strange behavior, and depends which type of file you are editing. Any idea on this?

Kind Regards.

**Comment by DavideViolante** [comment]
VS Code should have html intellisense out of the box. About Angular autocomplete you should check out the [extensions marketplace](https://marketplace.visualstudio.com/search?term=angular&target=VSCode&category=All%20categories&sortBy=Relevance)

**Comment by Thavarajan** [comment]
Try the path intellisense extension
